### DISCUSSION TITLE
docker compose ppm re-runs continously

### DISCUSSION THREAD

**Discussion by vasanth-mahendran** [meisnate12/Plex-Meta-Manager#1943] [Q&A] [answered]
Hello, 

I am new to docker and compose. i trying to set up ppm, plex using docker compose. everything runs succesfuly but i see that ppm re-runs continuously after it completes successfully. My assumption is ppm reruns only once a day at the default time 5:00

<img width="732" alt="image" src="https://github.com/meisnate12/Plex-Meta-Manager/assets/15712811/edb15bb9-4463-4fd5-b4a6-f66f9c94792d">

This is my docker compose file

ppm:
    container_name: ppm
    environment:
      - TZ=America/New_York
    user: 1000:1000
    image: meisnate12/plex-meta-manager
    depends_on:
      - plex
    restart: unless-stopped
    network_mode: host
    entrypoint:
      - python3
      - plex_meta_manager.py
    volumes:
      - ~/plex-meta-manager/config:/config:rw

**Comment by chazlarson** [comment, marked as answer]
It's the `--run` in the command line; that combined with the restart policy you have set is causing the endless loop.

**Reply by vasanth-mahendran** [reply]
Thanks got it working now. thanks
